Sass/style.css

```diff
@@ -0,0 +1,555 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: white;
+  font-family: "Lato", sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+}
+
+a {
+  text-decoration-line: none;
+}
+
+a:focus {
+  background-color: rgb(21, 19, 22);
+}
+
+.nav-bar {
+  background-color: rgb(246, 238, 238);
+  box-shadow: 0 2px 8px rgba(10, 9, 9, 0.26);
+  position: sticky;
+  display: block;
+  justify-content: space-between;
+  width: 100%;
+  min-height: 60px;
+}
+
+#nav-menu-cc {
+  border: 5px solid #ec5242;
+  padding: 8px;
+  width: 25%;
+  color: #ec5242;
+}
+
+.nav-bar .logo {
+  padding-left: 10px;
+  color: black;
+  font-weight: bolder;
+  font-size: 40px;
+}
+
+.nav-bar ul {
+  display: none;
+}
+
+.nav-bar button {
+  width: -moz-fit-content;
+  width: fit-content;
+  float: right;
+  border: none;
+  margin: 5px 10px 0 0;
+  background-color: transparent;
+}
+
+.courses img {
+  border-radius: 10px;
+  height: 60px;
+  width: 60px;
+  margin-bottom: 10px;
+}
+
+.nav-bar button img {
+  width: 50px;
+  height: 50px;
+}
+
+.home-page {
+  display: flex;
+  flex-direction: column;
+  min-height: 600px;
+  background-image: url(../images/main-background-img.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.home-page h1 {
+  color: #ec5242;
+  font-size: 50px;
+  font-family: "COCOGOOSE", sans-serif;
+  font-weight: 400;
+  text-transform: uppercase;
+}
+
+.home-page h1 strong {
+  color: #ec5242;
+  font-size: 40px;
+}
+
+.home-page h1 span {
+  color: #ec5242;
+  font-size: 20px;
+}
+
+.home-page .home-buttons-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
```

Comment:
Great job @degisew 🤝

- [ ] These buttons that you introduced in the design make your design so different from what is required. Please try to stick to the design given as much as possible
- Given design
![no-button](https://user-images.githubusercontent.com/63917725/183266950-3f2a81a5-ebd8-42f0-9b45-c96fa4f41e4f.PNG)
- Your design
![with buttons](https://user-images.githubusercontent.com/63917725/183266960-d7351535-92c6-44a7-a9e3-db6b42db21c6.PNG)